### ISSUE TITLE
Extra suffix on files on RestoreSSL

### ISSUE THREAD

**Issue by 6za** [kubefirst/kubefirst#191]
Logs: 
```
LOG: 2022/08/04 11:53:51.527125 /home/developer/app/internal/ssl/ssl.go:155: atlantis-tls.yaml
LOG: 2022/08/04 11:53:51.528020 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.528185 /home/developer/app/internal/ssl/ssl.go:155: atlantis-tls.yaml.clean
LOG: 2022/08/04 11:53:51.528979 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.529139 /home/developer/app/internal/ssl/ssl.go:155: atlantis-tls.yaml.clean.clean
LOG: 2022/08/04 11:53:51.529972 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.530137 /home/developer/app/internal/ssl/ssl.go:155: atlantis-tls.yaml.clean.clean.clean
LOG: 2022/08/04 11:53:51.530967 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.531129 /home/developer/app/internal/ssl/ssl.go:155: atlantis-tls.yaml.clean.clean.clean.clean
LOG: 2022/08/04 11:53:51.531892 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.532034 /home/developer/app/internal/ssl/ssl.go:155: chartmuseum-tls.yaml
LOG: 2022/08/04 11:53:51.532924 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.533082 /home/developer/app/internal/ssl/ssl.go:155: chartmuseum-tls.yaml.clean
LOG: 2022/08/04 11:53:51.533871 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.534018 /home/developer/app/internal/ssl/ssl.go:155: chartmuseum-tls.yaml.clean.clean
LOG: 2022/08/04 11:53:51.534848 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.535011 /home/developer/app/internal/ssl/ssl.go:155: chartmuseum-tls.yaml.clean.clean.clean
LOG: 2022/08/04 11:53:51.535895 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.536064 /home/developer/app/internal/ssl/ssl.go:155: chartmuseum-tls.yaml.clean.clean.clean.clean
LOG: 2022/08/04 11:53:51.536788 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.536996 /home/developer/app/internal/ssl/ssl.go:155: gitlab-tls.yaml
LOG: 2022/08/04 11:53:51.537945 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.538117 /home/developer/app/internal/ssl/ssl.go:155: gitlab-tls.yaml.clean
LOG: 2022/08/04 11:53:51.539011 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.539174 /home/developer/app/internal/ssl/ssl.go:155: gitlab-tls.yaml.clean.clean
LOG: 2022/08/04 11:53:51.540558 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.540968 /home/developer/app/internal/ssl/ssl.go:155: gitlab-tls.yaml.clean.clean.clean
LOG: 2022/08/04 11:53:51.541836 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.542057 /home/developer/app/internal/ssl/ssl.go:155: gitlab-tls.yaml.clean.clean.clean.clean
LOG: 2022/08/04 11:53:51.542959 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.543261 /home/developer/app/internal/ssl/ssl.go:155: gitlab-webservice-tls.yaml
LOG: 2022/08/04 11:53:51.544159 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.544398 /home/developer/app/internal/ssl/ssl.go:155: gitlab-webservice-tls.yaml.clean
LOG: 2022/08/04 11:53:51.545304 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.545533 /home/developer/app/internal/ssl/ssl.go:155: gitlab-webservice-tls.yaml.clean.clean
LOG: 2022/08/04 11:53:51.546421 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.546646 /home/developer/app/internal/ssl/ssl.go:155: gitlab-webservice-tls.yaml.clean.clean.clean
LOG: 2022/08/04 11:53:51.547545 /home/developer/app/internal/ssl/ssl.go:190: yaml cleaned written
LOG: 2022/08/04 11:53:51.547830 /home/developer/app/internal/ssl/ssl.go:155: gitlab-webservice-tls.yaml.clean.clean.clean.clean
```

**Comment by 6za** [comment]
On re-runs: `.clean` starts to repeat on the name of files.

**Comment by 6za** [comment]
Addressed on #223
